crates/matrix-sdk-base/src/rooms/members.rs

```diff
@@ -46,6 +46,11 @@ impl RoomMember {
         self.event.user_id()
     }
 
+    /// Get the original member event
+    pub fn event(&self) -> Arc<MemberEvent> {
+        self.event.clone()
+    }
```

Comment:
You may want to return an `&Arc<MemberEvent>`, then it's up to the user to decide if it's necessary to have an owned version of the `Arc<MemberEvent>` or not. It saves some costly operations from our side.

```suggestion
    pub fn event(&self) -> &Arc<MemberEvent> {
        &self.event
    }
```